pyproject.toml

```diff
@@ -12,11 +12,11 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.6.1,<3.10"
+python = ">=3.7.0,<3.10"
 python-magic = ">=0.4.12"
 attrs = ">=21.4"
 openpyxl = "^3.0.9"
-dcicutils = ">=3.11.0"
+dcicutils = ">=4.5.0"
```

Comment:
I think it's OK to make this change, but I point out that this is also an incompatible change only because it forces environmental differences where to run this you need to have environment variables set in a certain way you didn't used to. In the older version, not only did it lock to a particular set of dependency versions (which affected free-standing use and what environmental assumptions were made), but you had the option to include it as a library in a system that selected any subrange of your dependencies, so if you included it in a portal that required dcicutils `>=3.5,<4`, it would solve in the poetry solver. Doing this change means it will not. I might suggest leaving it still at `>=3.11.0`.  But I don't feel strongly about it and the concern is quite abstract. I think all the known consumers of this are OK using the version `>=4.5.0`. This one spec, though, would cause an incompatible change even if you weren't upgrading to drop Python 3.6 As it happens, since Python 3.6 is being dropped, it's an incompatible change anyway, so you're covered.